solidity/.gitignore

```diff
@@ -0,0 +1,11 @@
+# Compiler files
+cache/
+out/
+
+# Ignores development broadcast logs
+!/broadcast
+/broadcast/*/31337/
+/broadcast/**/dry-run/
```

Comment:
is this negation of broadcast `!/broadcast` needed if we later ignore those two paths?

Reply:
removed all this for now as we don't have the subdirectory. These lines were added automatically by forge. We're going to do integration tests with local-ic so doubt we are going have them later.